pdc/apps/release/serializers.py

```diff
@@ -134,17 +135,31 @@ class Meta:
                   'compose_set', 'integrated_with', 'sigkey', 'allow_buildroot_push',
                   'allowed_debuginfo_services', 'allowed_push_targets')
 
+    def set_default_sigkey(self, validated_data):
+        try:
+            if not validated_data["sigkey"]:
+                validated_data["sigkey"] = SigKey.objects.get(name=settings.RELEASE_DEFAULT_SIGKEY)
+        except AttributeError:
+            # If there is no Settings.RELEASE_DEFAULT_SIGKEY, raise it in server log
+            raise AttributeError('Settings.RELEASE_DEFAULT_SIGKEY does not exist, please set one".')
+        except Exception:
+            # If there is no the default SigKey object, just keep it as null.
+            pass
+        return validated_data
```

Comment:
If I understand correctly:
 * if the config option is missing, an error will be reported to users
 * if the config specifies a key that does not exist, nothing will happen

To me it this seems unintuitive: I would expect to be told when I make a typo in the configuration and the key does not exist. Also when the config is not given, it might work similarly to how it did before.

Reply:
Yep, correct, should change like this.